### PR TITLE
add doc link for insufficient privileges error

### PR DESCRIPTION
**Pull request by baurine**
How it looks like:

image

image

TODO:

- [ ] Add required permissions explanation to the pingcap docs repo

### CODE REVIEW

File: ui/dashboardApp/layout/translations/en.yaml [hunk 1]
```diff
@@ -2,6 +2,8 @@ signin:
   message:
     error: 'Sign in failed: {{ msg }}'
     success: Sign in successfully
+    access_doc_link: https://docs.pingcap.com/tidb/stable/dashboard-access
+    see_details: see details
```
Comment (breeswish) [inline ui/dashboardApp/layout/translations/en.yaml:6]: how about just calling it "Help"?

Comment (breeswish) [reply ui/dashboardApp/layout/translations/en.yaml:6]:
```suggestion
    access_doc: Help
```

**Review by breeswish — APPROVED** [review]

File: ui/dashboardApp/layout/translations/en.yaml (hunk 1 above)
Comment (baurine) [reply ui/dashboardApp/layout/translations/en.yaml:6]: ok

**Comment by shhdgit** [comment]
LGTM.

**Review by shhdgit — APPROVED** [review]

**Pull request merged**